### DISCUSSION TITLE
How to Export (and Import) an Environment JSON?

### DISCUSSION THREAD

**Discussion by sprive** [flawiddsouza/Restfox#55] [Q&A] [answered]
Hello,

Looking for a RestFox UI method to import/export Environment JSON. Is there such a thing? 

My workaround (which works great for "me"):

1.  Copy/Paste the JSON into and out of the RestFox Environment tab.
2. Use an external text editor to create your `example.env.json` file.

If this does not exist and is a feature request, LMK I will file an issue.
I am comfortable with the workaround, this ask is more about helping someone else migrate from Postman. 

Cheers

**Comment by flawiddsouza** [comment, marked as answer]
Hi. This feature doesn't yet exist but there's already a feature request for this at [export environment #40](https://github.com/flawiddsouza/Restfox/issues/40). I think we can club this into that. The mentioned issue only requests for exporting the environment along with the collection but we do need to have an import / export option in the environments popup.
